### ISSUE TITLE
Bandcamp requires User-Agent to access

### ISSUE THREAD

**Issue by tnachen** [Miserlou/SoundScrape#174]
Bandcamp return 403 without a proper User-Agent, so getting bandcamp metadata fails now.

**Comment by Miserlou** [comment]
Good one

**Comment by brachna** [comment]
True. Easy to fix using requests.session object. download_file already can work with it.

**Comment by Miserlou** [comment]
I also haven't able to repro this, needs example URL.

**Comment by brachna** [comment]
For example, https://jakob.bandcamp.com/album/solace-remastered

**Comment by Joshwoo70** [comment]
FYI Any updated browser would work.. such as chrome or firefox.

**Comment by Evolution0** [comment]
Any track, album, or Bandcamp CDN server url (popplers# -> p#.bcbits.com) will cause you to run into this issue now and yes using a User-Agent fixes it (Its how I got bandcamp-dl working again)

Otherwise you get a 403.

**Comment by tnachen** [comment]
I manually fixed it in my local soundscrape by setting a user agent when getting the metadata and downloading each mp3 link.

**Comment by Miserlou** [comment]
@tnachen - want to submit a PR? Thanks!
